webpack/main.cfg.js

```diff
@@ -0,0 +1,43 @@
+const webpack = require('webpack');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
+
+const { rules } = require('./rules');
+
+module.exports = function(env, argv) {
+  const IS_PRODUCTION = argv.mode === 'production';
+
+  /** @type {import('webpack').Configuration} */
+  const cfg = {
+    target: 'electron-main',
+    context: path.resolve(__dirname, '../src'),
+    entry: {
+      'electron-main': path.resolve(__dirname, '../src/app.js'),
+      'electron-preload': path.resolve(__dirname, '../src/preload.js'),
+    },
+    output: {
+      filename: '[name].js',
+    },
+    optimization: {
+      splitChunks: {
+        chunks: 'all',
+        name: 'main-vendor',
+      },
+    },
+    module: {
+      rules: [rules.node],
+    },
+    plugins: [
+      new CopyPlugin({
+        patterns: [{ from: 'icons', to: 'icons' }],
+      }),
+      new webpack.DefinePlugin({
+        IS_PRODUCTION: JSON.stringify(IS_PRODUCTION),
+      }),
+      // new BundleAnalyzerPlugin(),
```

Comment:
It may help you in the future. With `BundleAnalyzerPlugin` you can easily understand what package take too much space in a bundle. 
It's a rare usage case, so it doesn't worth adding any special flags. 

For example, moment.js have a lot of lang files for different locales. Since your app has a single language, you don't need them. So ignoring/excluding these lang files will save some space and can increase performance a bit.